canonical-web-and-design/ubuntu.com
# Update kubernetes nav

## Done

- Update kubernetes secondary nav 
- Add `Industry report` in the nav menu and create a redirect

![image](https://user-images.githubusercontent.com/57550290/165319494-1d59b11c-c187-4966-918f-eb02a8d8ae19.png)

- "Resources"(`/resources`) will be updated with this [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/11544)  

## QA

- Go to `/kubernetes`
- Check the nav items and redirect works.
- [Copy doc](https://docs.google.com/document/d/1ry3UzUJ5_danMmo1hjbI1cX6wfMY9PIlO8YWIDlw7OI/edit#heading=h.8apwcozc4g4d)

## Issue / Card

Fixes [#5091](https://github.com/canonical-web-and-design/web-squad/issues/5091)
